client/src/components/containersPanel.tsx

```diff
@@ -1,8 +1,9 @@
 import _ from 'lodash';
 import React, {Fragment} from 'react';
 import Field from './field';
+import {TODO} from "../utils/types";
```

Comment:
There should now be a `Container` type in `utils/types`. Would you mind switching this over to use that instead of the `TODO` type? It's not not fully fleshed out, but feel free to add any missing fields to `Container` until it compiles :)

Reply:
Hi Eric, I just found that `Container`  doesn't cover all the fields in the containersPanel and the same for others in the comments below. I guess it may be better to fix them one by one in separate PRs?

Reply:
Ok, let's go ahead and merge this ticket. We can circle back to convert these to more concrete types as we move forward.

That being said, please feel free to add missing fields to types like `Container`. Those are just stubbed out place holders with the idea that we'll add additional fields as we come across them :)